v6/cmd/common/logger.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"bytes"
+
+	"go.uber.org/zap"
```

Comment:
We were originally using "zap" in order to get structured logging for the interface with gcloud; however, I'm not sure how important that use case is at this point. I think this is fine to leave in place, but if you wanted to change it to use more standard tooling, that would probably also be fine.